.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
This seems to be a default configuration. Do we need this file?

Reply:
deleted